packages/web-react/src/types/heading.ts

```diff
@@ -0,0 +1,21 @@
+import { ElementType, JSXElementConstructor } from 'react';
+import { ChildrenProps } from './shared';
+
+export type TextBodySize = 'large' | 'medium' | 'small';
+export type HeadingSize = TextBodySize | 'xlarge' | 'xsmall';
```

Comment:
I'm not sure if this is intended relation or just a coincidence of overlapping scales…

Reply:
My intention is to define `shared/typography.ts` types after #228 will be merged and use `TextSize` as a default type extended by `xlarge` size.